src/tabs/tabs.js

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import * as React from 'react';
+import {getOverrides} from '../helpers/overrides.js';
+import Tab from './tab.js';
+import {
+  Root as StyledRoot,
+  TabBar as StyledTabBar,
+  TabContent as StyledTabContent,
+} from './styled-components.js';
+import type {TabsPropsT, SharedStylePropsArgT} from './types.js';
+import {ORIENTATION} from './constants.js';
+
+export default class StatefulTabs extends React.Component<TabsPropsT> {
+  static defaultProps: $Shape<TabsPropsT> = {
+    disabled: false,
+    onChange: () => {},
+    overrides: {},
+    orientation: ORIENTATION.horizontal,
+  };
+
+  onChange(activeKey: string) {
+    const {onChange} = this.props;
+    typeof onChange === 'function' && onChange({activeKey});
+  }
+
+  getTabs() {
+    const {activeKey, disabled, orientation, children} = this.props;
+    // eslint-disable-next-line flowtype/no-weak-types
+    const tabs = React.Children.map(children, (child: any, index) => {
+      if (!child) return;
+      const key = child.key || String(index);
+      const props = {
+        key,
+        id: key, // for aria-labelledby
+        active: key === activeKey,
+        disabled: disabled || child.props.disabled,
+        $orientation: orientation,
+        overrides: child.props.overrides,
+        onSelect: () => this.onChange(key),
```

Comment:
Let's change the arg passed to be an object: `this.onChange({key: key})` or `this.onChange({activeKey: key})`